README.rst

```diff
@@ -60,13 +81,17 @@ contributions!
 
 See the `developer docs`_ and the `contribution docs`_ for more information.
 
+
 Help
 ====
 
 Looking for more help?
 
-- Read `the project documentation`_
-- Check out our `support channels`_
+- Read `the project documentation`_.
+- Check out our `support channels`_.
+- For running CrateDB in the cloud, you are welcome to visit our offering
+  at `CrateDB Cloud`_ and its documentation at `CrateDB Cloud Tutorials`_.
```

Comment:
this is a little too wordy for this list, I think. how about something like:

- `CrateDB Cloud` is a fully-managed CrateDB *Database as a Service* (DBaaS) by `Crate.io`_

however, you might wanna check with a PM for phrasing. I expect we have a collection of pre-written sentences like this

Reply:
I've reworded the sentence according to your suggestion. Feel free to give advice for further adjustments.